site/static/compare.html

```diff
@@ -2,6 +2,7 @@
 <html>
 <head>
 <meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
It always worked fairly okay for me already on mobile, what sort of improvement does this bring?

Reply:
This is current page on mobile, we have to zoon in to view the page.
<details><summary>Old change</summary>

![Screen Shot 2020-10-09 at 09 40 55](https://user-images.githubusercontent.com/15225902/95535905-dcccd580-0a13-11eb-955a-ef1678357300.png)
</details>

With this meta tag, we don't. MDN wrote something about this: https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Responsive_Design#The_viewport_meta_tag